packages/@aws-cdk/aws-ec2/lib/launch-template.ts

```diff
@@ -68,6 +68,17 @@ export enum InstanceInitiatedShutdownBehavior {
   TERMINATE = 'terminate',
 };
 
+/**
+* Provide object for license specification
+* @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata.html#cfn-ec2-launchtemplate-launchtemplatedata-licensespecifications
+*/
+
+export interface ILicenseSpecifications {
```

Comment:
I think this PR should wait until we have an L2 for License Manager and the interface would then exist in that module. Until then users should use escape hatches to provide this value.